modules/alloy/src/main/ts/ephox/alloy/api/behaviour/Blocking.ts

```diff
@@ -0,0 +1,20 @@
+import * as BlockingApis from '../../behaviour/blocking/BlockingApis';
+import * as BlockingState from '../../behaviour/blocking/BlockingState';
+import { WrappedApiFunc } from '../../behaviour/common/Behaviour';
+import { AlloyBehaviour } from '../../behaviour/common/BehaviourTypes';
+import BlockingSchema, { BlockingConfig, BlockingConfigSpec } from '../../behaviour/blocking/BlockingSchema';
+import * as Behaviour from './Behaviour';
+
+export interface BlockingBehaviour extends AlloyBehaviour<BlockingConfigSpec, BlockingConfig, BlockingState.BlockingState> {
```

Comment:
This is extending the incorrect type I think? All other behaviours extend `Behaviour. AlloyBehaviour` as that's the public API type.

Reply:
fixed